analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LiteralsShouldNotBePassedAsLocalizedParameters.CSharp10.cs

```diff
@@ -1,9 +1,21 @@
-﻿public struct S
+﻿using System.ComponentModel;
+
+public struct S
 {
+    [Localizable(true)]
     public string Property { get; set; }
 
     public void M()
     {
-        (Property, var b) = ("a", "B"); // FN
+        (Property, var b) = ("a", "B");  // Noncompliant
+        //                   ^^^
+
+        (Property, Property) = ("a",   // Noncompliant
```

Comment:
[Docs](https://github.com/SonarSource/sonar-dotnet/blob/d56b3718f2dc350e9427dbc51af9d514a8e6c7c6/analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/IssueLocationCollector.cs#L81) for when having to declare multiple issues per line.